app/uk/gov/hmrc/rasapi/repository/GridFsTTLIndexing.scala

```diff
@@ -34,55 +33,46 @@ trait GridFsTTLIndexing {
   private lazy val OptExpireAfterSeconds = "expireAfterSeconds"
   protected lazy val UploadDate = "uploadDate"
 
-  def addAllTTLs(gfs: GridFS[BSONSerializationPack.type])(implicit ec: ExecutionContext): Future[Seq[Boolean]] = {
-    Future.sequence(Seq(
-      addFilesTTL(gfs),
-      addChunksTTL(gfs)
-    ))
+  private def listAllIndexes(mdb: MongoDatabase, collectionName: String): Seq[Document] = {
+    log.info(s"[GridFsTTLIndexing][retrieveLastUpdatedIndex] Searching $collectionName for all indexes.")
+    Await.result(mdb.getCollection(collectionName)
```

Comment:
I think this Await is unnecessary, since you can return a Future[Seq[Document]] from this function and .map on it in checkAndEnsureTTL?